sources/platform/actors/publishing/monetize.mdx

```diff
@@ -13,13 +13,131 @@ Apify Store allows you to monetize your web scraping and automation projects by
 
 ## Pricing models
 
-Actors in the Apify Store can be published under one of the following pricing models:
+Actors in Apify Store can be published under one of the following pricing models:
 
-1. **Free**: Users can run the Actor without any additional charges beyond the platform usage costs generated by the Actor
-2. **Rental**: Users pay for the platform usage costs. However, after a trial period, they need to pay a flat monthly fee to the develop to continue using the Actor
-3. **Pay per Result**: Users don't pay for the platform usage costs. Instead, they pay the developer based on the number of results produced by the Actor.
-4. **Pay per Event**: Users don't pay for the platform usage cost the Actor generates. Instead, they pay based on specific events defined by the developer, such as generating a single result or initiating the Actor.
+1. **Free**: Users can run the Actor without any additional charges beyond the platform usage costs generated by the Actor.
+2. **Rental**: Users pay for the platform usage costs. However, after a trial period, they need to pay a flat monthly fee to the developer to continue using the Actor.
+3. **Pay per result**: Users don't pay for the platform usage costs. Instead, they pay the developer based on the number of results produced by the Actor.
+4. **Pay per event**: Users don't pay for the platform usage cost the Actor generates. Instead, they pay based on specific events defined by the developer, such as generating a single result or starting the Actor.
```

Comment:
Suggestion: Please, add that the events are triggered programatically from the Actor source code.